conf/airframes/tudelft/rot_wing_25kg.xml

```diff
@@ -156,9 +157,9 @@
     </commands>
     
 
-    <auto_rc_commands>
-        <set VALUE="@AUX4" COMMAND="THRUST_X"/>
-    </auto_rc_commands>
+    <rc_commands>
+        <set COMMAND="THRUST_X" VALUE="@AUX4"/>
+    </rc_commands>
```

Comment:
Correct, with the proposed solution this is not needed. I did not remove it yet only because I was not convinced. What I wanted is really ```rc_command```: as in when you are in a manual mode just copy the stick to the actuator. That is exactly what is needed, but with ```stabilization_indi```, that does not work (since stabilization writes all actuators before the main loop copies commands to actuators).